app/src/main/java/com/example/henzoshimada/feeltrip/FeelTripApplication.java

```diff
@@ -18,6 +20,26 @@
     to the queue.
      */
 public class FeelTripApplication extends Application {
+
+    private static final List<String> allEmotions = Arrays.asList( // MODIFY THIS IF YOU WISH TO ADD/DELETE EMOTIONS
+            "Zero", // THIS IS HERE SO WE CAN REFERENCE THE EMOTIONS WITHOUT WORRYING ABOUT ZERO INDEXING
+            "Angry",
+            "Confused",
```

Comment:
Editing this simple List allows us to Add/Delete new emotions at our own discretion. This is the EZ LOAD for emotions.